chronos/transports/stream.nim

```diff
@@ -1063,11 +1066,18 @@ when defined(windows):
               # Start tracking transport
               trackStream(ntransp)
               retFuture.complete(ntransp)
-          elif int32(ovl.data.errCode) == ERROR_OPERATION_ABORTED:
+          of OSErrorCode(ERROR_OPERATION_ABORTED):
             # CancelIO() interrupt or close.
             server.asock.closeSocket()
             retFuture.fail(getServerUseClosedError())
             server.clean()
+          of OsErrorCode(common.WSAENETDOWN), OSErrorCode(common.WSAENETRESET),
+             OSErrorCode(common.WSAECONNABORTED),
+             OSErrorCode(common.WSAECONNRESET),
+             OSErrorCode(common.WSAETIMEDOUT):
+            server.asock.closeSocket()
+            retFuture.fail(getConnectionAbortedError(int(ovl.data.errCode)))
+            server.clean()
```

Comment:
Seems okay, I'm not familiar enough with Windows networking to verify correctness though.